### ISSUE TITLE
Add logo

### ISSUE THREAD

**Issue by cpurta** [cpurta/tatanka#3]
We need to add a cool logo for this project. Just need a png file added to the /assets folder name logo.png. I am open to any type of design.

**Comment by sparsh-99** [comment]
Hey @cpurta , I am happy to contribute. Let's discuss what type of logo you wanted!

**Comment by Catnip-01** [comment]
hi there, i'm new to github, but i have all the time in the world, so i guess i'm in !
plus, my mum's an artist, so i think that might help further !
again, i'd need details about your project, so that i can see what theme might suit your project.

**Comment by lakshya-dhariwal** [comment]
I can make the logo fast.  What do you want . What is the colour scheme . What is the topic? Can you assign it to me ?

**Comment by cpurta** [comment]
The project is name Tatanka which is Lakota for Buffalo/Bison. Since this program is used to analyze cryptocurrency markets and attempt to find "bullish" trend that was inspiration for the name. So I was thinking of using a bison for the logo. I am thinking of something similar to these but using a Bison instead of a bull:

![BullJumpingLogo](https://user-images.githubusercontent.com/7330964/94734139-a8735e00-031d-11eb-9a5f-25d181d33562.png)

![ToughBullLogo](https://user-images.githubusercontent.com/7330964/94734148-ac06e500-031d-11eb-932a-450b736e0251.jpg)

As far as the color scheme I was thinking something darker, black or a darker blue. But I am open to perhaps some lighter colors as well. 

Let me know if you have any more questions.

**Comment by cpurta** [comment]
@sparsh-99 @LySergicaciD125 @LilAc1153 does my last comment address the questions you guys have? If so I am happy to assign it over to anyone wanting to take it on.

**Comment by kavita121** [comment]
Heyy!! Can I work on this??

**Comment by cpurta** [comment]
@kavita121 you sure can. I'll assign it over to you since no one else has responded to my last update.

**Comment by kavita121** [comment]
Okay!! I will try :)

**Comment by Catnip-01** [comment]
![image](https://user-images.githubusercontent.com/72100668/95431209-750e8000-096a-11eb-8f85-070159a4970e.png)

**Comment by Catnip-01** [comment]
so this is just a very rough idea, we can maybe give it a lot more corners and edges, i was thinking we could avoid the curves altogether, and then again comes the question of how you'd like the horns to be.

**Comment by Catnip-01** [comment]
ik that look like a third grader's artwork, but it's nowhere close to done, so maybe keep that in mind ?

**Comment by LilAc1153** [comment]
![Uploading image.png…]()
something like this...

**Comment by Catnip-01** [comment]
![image](https://user-images.githubusercontent.com/72100668/95432354-2bbf3000-096c-11eb-82dc-0d61a3300c2e.png)
something like this maybe...
i'll upload a colored version in a while.

**Comment by LilAc1153** [comment]
![Uploading image.png…]()
that's with an outline..

**Comment by Catnip-01** [comment]
![image](https://user-images.githubusercontent.com/72100668/95432569-780a7000-096c-11eb-92e1-c00af4dd8c99.png)
that's with the outline...

**Comment by Catnip-01** [comment]
![image](https://user-images.githubusercontent.com/72100668/95433914-3975b500-096e-11eb-8b37-1b8c1b7dca79.png)
i'll be uploading a bunch of different color versions, and you can choose the most suitable one.

**Comment by Catnip-01** [comment]
![image](https://user-images.githubusercontent.com/72100668/95434069-72ae2500-096e-11eb-80cf-f04119dc7bf4.png)

**Comment by Catnip-01** [comment]
![image](https://user-images.githubusercontent.com/72100668/95434376-cb7dbd80-096e-11eb-9d0a-c4d4f7d7f57b.png)
i have no idea what to do about the remaining horn and the eyebrows, i remember you mentioning the use of blue somewhere, but then adain blue and yellow isn't something i like, but a darker blue might do the trick for the eyebrows... about the horn, it's up to you.

**Comment by Catnip-01** [comment]
oh, and this is just a trial, the eyebrows are messed up, i agree, but then again i'm going to make a better one if we go ahead with it.

**Comment by cpurta** [comment]
Yeah I think that could work. If you get a png or svg of it feel free to open up a PR.

**Comment by Catnip-01** [comment]
![colored](https://user-images.githubusercontent.com/72100668/95651708-c608aa80-0b09-11eb-8708-b24c30d255ce.png)
here's the colored version, i'll put up a colorless version too, in case you wanna test some colors yourself.

**Comment by Catnip-01** [comment]
![logo_colorless](https://user-images.githubusercontent.com/72100668/95651820-65c63880-0b0a-11eb-9151-492c72c23569.png)
here's the colorless version.

**Comment by Catnip-01** [comment]
i have all the intermediate versions of the colors, so in case you need them in future, just let me knw, and i'll convert them into PNGs and upload them.
